aten/src/ATen/core/dispatch/Dispatcher.cpp

```diff
@@ -254,6 +255,66 @@ void Dispatcher::deregisterDef_(
   cleanup(op, op_name);
 }
 
+namespace {
+
+using AbstractImplPyStubsType = std::unordered_map<at::OperatorName, const char*>;
```

Comment:
I think most of `c10` uses `ska::flat_hash_map` over `unordered_map`, although perf probably doesn't matter a ton here.